R/understat_match_players.R

```diff
@@ -0,0 +1,53 @@
+
+#' Get Understat match player data
+#'
+#' Returns player values for a selected match from Understat.com
+#'
+#' @param match_url the URL of the match played
+#'
+#' @return returns a dataframe with data for all players for the match
+#'
+#' @importFrom magrittr %>%
+#'
+#' @export
+
+understat_match_players <- function(match_url) {
```

Comment:
Please indent the body of this function (add 2 spaces at the beginning of each line). Technically it's not wrong R syntax and we are pretty relaxed with spacing/indentation rules, but that is fairly unconventional.